packages/admin/admin/src/mui/MasterLayout.tsx

```diff
@@ -1,14 +1,59 @@
-import { ComponentsOverrides, CssBaseline, Theme } from "@mui/material";
-import { WithStyles, withStyles } from "@mui/styles";
+import { ComponentsOverrides, CssBaseline } from "@mui/material";
+import { css, styled, Theme, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 
 import { AppHeader } from "../appHeader/AppHeader";
 import { AppHeaderMenuButton } from "../appHeader/menuButton/AppHeaderMenuButton";
-import { MasterLayoutClassKey, styles } from "./MasterLayout.styles";
 import { MasterLayoutContext } from "./MasterLayoutContext";
 import { MenuContext } from "./menu/Context";
 
-export interface MasterLayoutProps {
+export type MasterLayoutClassKey = "root" | "header" | "contentWrapper";
+
+const Root = styled("div", {
+    name: "CometAdminMasterLayout",
+    slot: "root",
+    overridesResolver(_, styles) {
+        return [styles.root];
+    },
+})(
+    () => css`
+        display: flex;
+        flex-wrap: nowrap;
+    `,
+);
+
+const Header = styled("div", {
+    name: "CometAdminMasterLayout",
+    slot: "header",
+    overridesResolver(_, styles) {
+        return [styles.header];
+    },
+})(
+    ({ theme }) => css`
+        z-index: ${theme.zIndex.drawer} - 10;
```

Comment:
Does this work? I think this would be the correct syntax:

```suggestion
        z-index: ${theme.zIndex.drawer - 10};
```